fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -499,6 +590,13 @@ protected void runRunningJob() throws AlterCancelException {
             LOG.info("schema change tasks not finished. job: {}", jobId);
             List<AgentTask> tasks = schemaChangeBatchTask.getUnfinishedTasks(2000);
             for (AgentTask task : tasks) {
+                if (invertedIndexChange) {
+                    // TODO(wy): more elegant implementation
```

Comment:
Is this TODO must be fixed?

Reply:
yes, need to do, but not in this pr, "more elegant implementation" need implement specify partition to rebuild index data